jenkins/jobs/bml_integration_tests.pipeline

```diff
@@ -0,0 +1,100 @@
+import java.text.SimpleDateFormat
+
+ci_git_credential_id = "metal3-jenkins-github-token"
+
+def CLEAN_TIMEOUT = 600
+def TIMEOUT = 5400
+
+if (env.TESTS_FOR) {
+  if ( (env.TESTS_FOR).startsWith('feature_tests') ) {
+    TIMEOUT = 12600
+  }
+}
```

Comment:
Do we need this for bm_lab? If yes, not sure if 3,5h would be enough to finish current number features tested in the tests.

Reply:
Hm, it's a good point, I expect we won't want to time out at all.